discordbot/README.md

```diff
@@ -52,6 +52,21 @@ https://discord.com/oauth2/authorize?client_id=CLIENTID&scope=bot, where **CLIEN
 
 **NOTE:** Registering is free for all open source projects and if your discord bot uses fewer than 1250 uploads per day.
 
+### HEROKU
+
+1. Add the following lines into requirements.txt
+    1. discord==1.7.3

```

Comment:
In fact these already belong to initial installation some lines above, so no need to repeat

Reply:
Forgot to change that.